src/ocean/util/app/ext/TimerExt.d

```diff
@@ -272,7 +277,14 @@ public class TimerExt : IApplicationExtension
         auto reregister = event.dg();
         if ( reregister )
         {
-            this.registerMicrosec(event.dg, event.repeat_microsec, event.repeat_microsec);
+            do
+            {
+                event.next_call += event.repeat_microsec;
+            }
+            while ( event.next_call < MicrosecondsClock.now_us() );
```

Comment:
It's probably worth mentioning this behaviour in the method doc and the release notes.